### PR TITLE
add new 'cloud' variable

### PR DESCRIPTION
**Pull request by johanneskastl**
* add new variable `openstack_cloud`
* smaller cleanups in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,11 +13,18 @@ Kubernetes on OpenStack with the help of Terraform and RancherKubernetesEngine (
 Using `clouds.yaml`:
 
 ```shell
-export OS_CLOUD=<section>
+export TF_VAR_openstack_cloud=<section>
```
Comment (curx) [inline README.md:16]: we should let this option and add another possibility using `TF_VAR_`

File: variables.tf [hunk 2]
```diff
@@ -33,6 +33,10 @@ variable "availability_zone_hints_network" {
   default = ["nova"]
 }
 
+variable "openstack_cloud" {
+  description = "Which cloud to use from clouds.yml"
+}
```
Comment (curx) [inline variables.tf:44]: we should add a type for this var also

**Review by curx — CHANGES_REQUESTED** [review]
@johanneskastl can you please take a look? (PTAL)

File: variables.tf (hunk 2 above)
Comment (johanneskastl) [reply variables.tf:44]: Fixed.

File: README.md (hunk 1 above)
Comment (johanneskastl) [reply README.md:16]: Actually, I think we can have the best of both ways:
`export TF_VAR_openstack_cloud=$OS_CLOUD`

If you need/want to set variables via shell, then you can set this to either a string or just use the content of the `OS_CLOUD` variable (if it is defined).

But I would like to give users the possibility to not need environment variables at all, but rather have everything in  terraform.tfvars. (Which I find is a lot less error prone when switching between shells and projects).

I already changed the openrc.sh example accordingly.

**Comment by grawert** [comment]
Fixed by 1a4966649dbad6aba054b32d7fee357a3c70e952

**Pull request closed without merging**